addon/styles/_frost-expand.scss

```diff
@@ -19,14 +19,22 @@ $frost-color-expand-content-text: $frost-color-grey-3 !global;
     }
   }
 
+  .pointer {
```

Comment:
Yes, it's because of the new DOM. When `isChevronOnlyClickTrigger` is set to `true`, the entire label should not have the pointer cursor so this class is added to the label only when the value is `false`.